### DISCUSSION TITLE
Can't get the nextcloud-aio-nextcloud container to start

### DISCUSSION THREAD

**Discussion by Tresillo2017** [nextcloud/all-in-one#3434] [Questions] [answered]
### Steps to reproduce
1. Install nextcloud-aio
2. Start it

### Expected behavior 
To start normally without any problem
### Actual behavior 
Doesn't start because of folder permision which I already gave it.

### Host OS 
Ubuntu 23

#### Nextcloud AIO version 
27.0.1
#### Current channel 
Stable
#### Other valuable info 
![image](https://github.com/nextcloud/all-in-one/assets/56476330/5a09008b-b7c4-4d40-b037-40f8cef50ed0)

> I'm trying to migrate from nextcloud docker to aio. All my data was on a nas which is mounted onto `/mnt/nas`, but I changed it to `/mnt/ncdata`.

**Comment by szaimen** [comment]
> Doesn't start because of folder permision which I already gave it.

You found the problem then

**Reply by Tresillo2017** [reply]
But I can't start it even thought I run 'chmod - R 33:0' and 'chown -R 750'. It won't start.

**Reply by szaimen** [reply]
Can you follow https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance and try again?

**Reply by Tresillo2017** [reply]
It worked but it's giving me an error.
![image](https://github.com/nextcloud/all-in-one/assets/56476330/273e59fa-b441-4de5-9331-3099890620af)

**Reply by szaimen** [reply]
Please follow all steps. Did you also delete the files in the chosen NEXTCLOUD_DATADIR ?

**Reply by Tresillo2017** [reply]
I didn't delete the files on the nextcloud data dir, since I want to keep them, they are in a nas mounted at `/mnt/ncdata`

**Reply by szaimen** [reply, marked as answer]
I see, this is not going to work then. Please follow https://github.com/nextcloud/all-in-one/blob/main/migration.md

**Reply by Tresillo2017** [reply]
Deleted the data on the folder, and only kept appdata, and it's giving me the same error.

**Reply by szaimen** [reply]
Hi, did you see the migration guide?

**Reply by Tresillo2017** [reply]
That it's was I was following, and redone it again, but it's the same error. I'm trying again from 0
